src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/UploadQuotaType.php

```diff
@@ -48,7 +48,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add(
                 self::FIELD_MAX_SIZE_ATTACHED_FILES,
-                TextWithUnitType::class,
+                NumberWithUnitType::class,
```

Comment:
Why not use `IntegerWithUnitType` here? Since it appears the value is an `int` no?
Same for other fields in this form type (and maybe other places in the PR I didn't check each field), when the value is an integer `IntegerWithUnitType` seems more adapted